urllib3/connectionpool.py

```diff
@@ -448,7 +448,8 @@ def urlopen(self, method, url, body=None, headers=None, retries=None,
         """
         Get a connection from the pool and perform an HTTP request. This is the
         lowest level call for making a request, so you'll need to specify all
-        the raw details.
+        the raw details. To make HTTP calls, use :func:`urllib3.request.RequestMethods.request`
+        instead.
```

Comment:
This reads weirdly to me. Maybe: "In general `request` should be preferred to this method."?